engines/native/sandbox.go

```diff
@@ -170,16 +188,18 @@ func (s *sandbox) waitForTermination() {
 
 	s.resolve.Do(func() {
 		// Halt all other sub-processes
-		system.KillByOwner(s.user)
+		if s.engine.config.CreateUser {
+			system.KillByOwner(s.user)
+		}
```

Comment:
Again, don't we need an alternative here for the other case? If we kill the process tree... I guess the process might no longer exist, or?